### ISSUE TITLE
`@finos/perspective` 3.1.0 RuntimeError: memory access out of bounds

### ISSUE THREAD

**Issue by rajsite** [finos/perspective#2796]
### Steps to Reproduce:

Trying to update the [vite example](https://github.com/ProspectiveCo/perspective-vite-example) to perspective 3.1.0 by switching to the `inline.js` files to avoid needing extra bundler configuration.

However when run the example now gives the following error message:
```
009593ea:0x1f6955 Uncaught RuntimeError: memory access out of bounds
    at 009593ea:0x1f6955
    at 009593ea:0x15e233
    at 009593ea:0x1e3c35
    at 009593ea:0x1f796d
    at 009593ea:0x112c07
    at 009593ea:0x25f89
    at 009593ea:0x5cb41
    at 009593ea:0x9d125
    at 009593ea:0xb3adf
    at 009593ea:0x184108
$func8408	@	009593ea:0x1f6955
$func3703	@	009593ea:0x15e233
$func7771	@	009593ea:0x1e3c35
$func8604	@	009593ea:0x1f796d
$func2019	@	009593ea:0x112c07
$func324	@	009593ea:0x25f89
$func383	@	009593ea:0x5cb41
$func615	@	009593ea:0x9d125
$func752	@	009593ea:0xb3adf
$__wbindgen_export_8	@	009593ea:0x184108
We	@	@finos_perspective-v…s.js?v=385eb104:252
s	@	@finos_perspective-v…s.js?v=385eb104:202
```

See stackblitz example based: https://stackblitz.com/edit/perspective-vite-memory-access-runtime-error?file=main.js

I get the same errors going as bare bones as possible. Just serving JS and HTML without a build step using the `inline` builds for perspective-viewer and perspective and the `cdn` build for perspective-viewer-datagrid, see: https://stackblitz.com/edit/perspective-inline-runtime-error?file=main.js

### Expected Result:

Perspective example is able to run and show the `perspective-viewer` element.

### Actual Result:

See above

-   Win 10

If you are reporting a UX/UI error:

N/A

If you are reporting a library error:

N/A

If you are reporting a build or install error:

done, see above.

### Environment:

For JavaScript (browser):

-   `@finos/perspective` version: 3.1.0
-   Browser and version Chrome 129.0.6668.101
-   OS Win 10
-   (if websocket) Language/version/OS of perspective server N/A

For Node.js:
N/A

-   `node` version
-   OS

For Python
N/A

-   `python` interpreter version (Only CPython).
-   package manager and version (conda/pip/\*)
    -   Are you compiling from an sdist of wheel?
-   Platform and version (Jupyter/tornado/lib/\*)
-   OS

### Additional Context:

N/A

**Comment by rajsite** [comment]
@texodus does it look like I'm leveraging the inline build as intended?

I created two simple jsbin examples (that are not using a bundler): one for [inline build](https://jsbin.com/jasorehopi/2/edit?html,output) and one for [cdn build](https://jsbin.com/sumavuw/1/edit?html,output).

The cdn example does seem to work correctly but the inline example shows the error from the issue.

**Comment by rajsite** [comment]
@texodus The inline build fixes included in v3.1.2 https://github.com/finos/perspective/releases/tag/v3.1.2 seem to work for me!

Updated stackblitz with vite using inline builds: https://stackblitz.com/edit/perspective312-vite?file=README.md
Updated jsbin using inline builds via unpkg: https://jsbin.com/buwicow/1/edit?html,output

I'll go ahead and close this issue, thanks!
